### PR TITLE
feat: replace payment dummy ui with web3 payment

### PR DESCRIPTION
**Pull request by techyNonso**
### Description

- [x] Resolves 

### Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [x] I have added suitable unit tests (if applicable).

To be reviewed by: @Nasfame @techyNonso

### CODE REVIEW

**Comment by techyNonso** [comment]
You can merge but note, flow is not complete

File: hardhat/contracts/sample/Payment.sol [hunk 1]
```diff
@@ -1,41 +1,41 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity ^0.8.22;
+pragma solidity ^0.8.9;
```
Comment (Nasfame) [inline hardhat/contracts/sample/Payment.sol:3]: @techyNonso the solidity version should be 0.8.22

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: hardhat/contracts/sample/Payment.sol
Comment (Nasfame) [inline hardhat/contracts/sample/Payment.sol:1]: can add the contracts related to the app to hardhat/contracts. If its the sample contracts then copy it to hardhat/contracts

**Review by Nasfame — CHANGES_REQUESTED** [review]
@techyNonso

**Review by Nasfame — APPROVED** [review]
LGTM, untested

**Comment by Nasfame** [comment]
@techyNonso make if any changes left

**Pull request merged**